utils/build.ps1

```diff
@@ -2745,6 +2748,13 @@ try {
 Fetch-Dependencies
 
 if (-not $SkipBuild) {
+  if (-Not (Test-CMakeAtLeast -Major 3 -Minor 28)) {
+    throw "Minimum required CMake version is 3.28"
```

Comment:
CMake 3.28+ is now required globally: https://github.com/swiftlang/swift/blob/main/tools/swift-inspect/CMakeLists.txt#L1